snomed/com.b2international.snowowl.snomed.ql/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,35 @@
+Manifest-Version: 1.0
```

Comment:
This project is still missing:
- a `pom.xml` file and `module` entry in the corresponding aggregator pom
- a feature.xml entry in `snomed.core.feature`